### ISSUE TITLE
[QUESTION] why do you need to get the minnt and oldsrc files yourself if opennt is dead

### ISSUE THREAD

**Issue by GiladDaKing** [leecher1337/ntvdmx64#284]

**Comment by GiladDaKing** [comment]
also the win7 iso

**Comment by leecher1337** [comment]
Because they are an essential requirement and contain the necessary sourcecode and build environment.
Due to legal reasons, it is impossible to put them up into this repository, as it would get then DMCAed very soon. I am legally not allowed to redistribute copyrighted material, therefore the user needs to get hold of the material on his own and it's user's sole resposibility what he does with it.

**Comment by GiladDaKing** [comment]
so cant you give a link?

**Comment by leecher1337** [comment]
No, I am not allowed to link to copyrighted material, this would make the repository tainted. 
Google is your friend, there are some sites hosting required material.
There is a hint for opennt without any links in issue 163 and there is also a fork of this repo where someone made a CI-script that contains links, but this repository has to stay clean.

**Comment by GiladDaKing** [comment]
> also the win7 iso

and what with this one. why you have to download it manually (from the link) and you cant download it automatically

**Comment by GiladDaKing** [comment]
also this website has minnt https://www.planetkode.com/2021/05/

**Comment by leecher1337** [comment]
Congratulations, you are able to use google like the documentation told you to do!
The winfuture-site for win 7 referenced in autobuild script i.e. doesn't allow direct download.

**Comment by GiladDaKing** [comment]
what do you mean by "doesn't allow direct download"

**Comment by GiladDaKing** [comment]
i have the link for downloading the win7 iso

**Comment by GiladDaKing** [comment]
> Congratulations, you are able to use google like the documentation told you to do! The winfuture-site for win 7 referenced in autobuild script i.e. doesn't allow direct download.

the link for downloading it is https://dl2-new.winfuture.de/9YcstEhXdBSiIldeSaOqvw/1732678506/3291/ssd/Betriebssysteme/windows7/de_windows_7_professional_with_sp1_x86_dvd_u_677093.iso

**Comment by GiladDaKing** [comment]
> Congratulations, you are able to use google like the documentation told you to do! The winfuture-site for win 7 referenced in autobuild script i.e. doesn't allow direct download.

the link is https://dl2-new.winfuture.de/7ajn1pZCSXXZnVJVt2DYiA/1732815064/3291/ssd/Betriebssysteme/windows7/de_windows_7_professional_with_sp1_x86_dvd_u_677093.iso

**Comment by GiladDaKing** [comment]
so that what it mean

**Comment by GiladDaKing** [comment]
but why do you need this win7 iso and not other one from another website?

**Comment by leecher1337** [comment]
Feel free to propose another one that is freely available and can be downloaded fast.

**Comment by GiladDaKing** [comment]
this is the same file but from another website -> https://drive.massgrave.dev/de_windows_7_professional_with_sp1_x86_dvd_u_677093.iso

**Comment by GiladDaKing** [comment]
for me it download at approximately 4 minutes

**Comment by leecher1337** [comment]
massgrave.dev links also do expire and so are also no direct download url (i.e. I get: ⌛ The download link will expire 2024-12-23 00:54:24 UTC )
